app/src/main/java/com/sdp/movemeet/DistanceCalculator.java

```diff
@@ -1,68 +1,130 @@
 package com.sdp.movemeet;
 
 import android.os.Build;
+
 import androidx.annotation.RequiresApi;
+
 import com.sdp.movemeet.Activity.Activity;
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class DistanceCalculator {
-    private final Double EARTH_RADIUS = 6.3781 * Math.pow(10, 3);
+    // private final Double EARTH_RADIUS = 6.3781 * Math.pow(10, 3);
+
+    // This constant is more ~canonical~
+    private final Double EARTH_RADIUS = 6.371 * Math.pow(10, 3);
```

Comment:
Good point to use the formula used by everyone!

Reply:
Thanks!